### PR TITLE
Update retry docs.

### LINKED ISSUE #1694: Update "Configuring Retries" page for v2.11 changes
https://linkerd.io/2.14/tasks/configuring-retries/

This page doesn't appear to have been updated for the v2.11 change to allow retries for HTTP routes with bodies and gRPC.

### PR DESCRIPTION
**Pull request by kflynn**
Fixes #1694

Signed-off-by: Flynn <flynn@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/2-edge/tasks/configuring-retries.md [hunk 1]
```diff
@@ -47,6 +47,10 @@ spec:
     isRetryable: true ### ADD THIS LINE ###
 ```
 
+Retries are supported for _all_ idempotent requests, whatever verb they use
+and whether or not they have bodies. In particular, this means that gRPC
+requests can be retried.
```
Comment (hawkw) [inline linkerd.io/content/2-edge/tasks/configuring-retries.md:52]: we should probably mention the body length limit here, though?

**Review by hawkw — COMMENTED** [review]
overall, this looks good to me; i think that if we're updating the documentation to reflect that requests with bodies are retryable, we should document the length limit for retryable bodies (64KB; see https://github.com/linkerd/linkerd2-proxy/blob/e62cc28ffe59ba9d1906b04a6873bd95ef23a2f7/linkerd/app/outbound/src/http/retry.rs#L41-L42)

**Review by wmorgan — APPROVED** [review]
ty

File: linkerd.io/content/2-edge/tasks/configuring-retries.md (hunk 1 above)
Comment (kflynn) [reply linkerd.io/content/2-edge/tasks/configuring-retries.md:52]: Excellent point. 🙂

File: linkerd.io/content/2.11/tasks/configuring-retries.md [hunk 2]
```diff
@@ -39,6 +39,11 @@ spec:
     isRetryable: true ### ADD THIS LINE ###
 ```
 
+Retries are supported for _all_ idempotent requests, whatever verb they use,
+and whether or not they have a body. In particular, this mean that gRPC
+requests can be retried. However, requests will not be retried if the body
+exceeds 64KiB.
```
Comment (hawkw) [inline linkerd.io/content/2.11/tasks/configuring-retries.md:45]: do you think we ought to link to the blog post on retry bodies here?

**Review by hawkw — APPROVED** [review]
lgtm, one last suggestion

File: linkerd.io/content/2.11/tasks/configuring-retries.md (hunk 2 above)
Comment (kflynn) [reply linkerd.io/content/2.11/tasks/configuring-retries.md:45]: That's an excellent idea! https://linkerd.io/2021/10/26/how-linkerd-retries-http-requests-with-bodies/ correct?

Comment (hawkw) [reply linkerd.io/content/2.11/tasks/configuring-retries.md]: yup!

**Review by hawkw — APPROVED** [review]
looks great!

**Pull request merged**